lib/ukboot/Config.uk

```diff
@@ -77,6 +77,10 @@ if LIBUKBOOT
 	int "Maximum number of arguments (max. size of argv)"
 	default 60
 
+	config LIBUKBOOT_DISABLE_WEAK_MAIN
+	bool
```

Comment:
```suggestion
	bool "Disable weak main"
```

Reply:
It doesn't show up in the menu if it's empty.

Reply:
Yes, that was the intention. The plan is to have it as an internal configuration that is selected by applications via `select LIBUKBOOT_DISABLE_WEAK_MAIN`.